charts/fluentd-elasticsearch/values.yaml

```diff
@@ -49,6 +49,10 @@ elasticsearch:
   buffer_queue_limit: 8
   logstash_prefix: 'logstash'
 
+# If you want to change args of fluentd process
+# by example you can add -vv to launch with trace log
+fluentd_args: "fluentd--no-supervisor -q"
```

Comment:
Camelcase should be used -> fluentdArgs.
Value should be "--no-supervisor -q".

Reply:
I do it